src/app/_hooks/useFilterOptionsWithCount.ts

```diff
@@ -40,11 +40,11 @@ export function useCategoryOptionsWithCount<Entry extends Object>({
   const optionsWithAllAndCount = useMemo(() => {
     const allOptionWithCount = {
       ...allOption,
-      count: getTotalCategoryCount(optionsWithCount),
+      count: sumObjectCounts(optionsWithCount),
     }
 
     return [allOptionWithCount, ...optionsWithCount]
   }, [allOption, optionsWithCount])
 
-  return { categoryOptionsWithCount: optionsWithAllAndCount }
+  return optionsWithAllAndCount
```

Comment:
I find quite a few namings in this hook confusing - Perhaps we can pair and find some clarity.